packs/csv/tests/test_action_parse.py

```diff
@@ -0,0 +1,35 @@
+import unittest2
+
+from parse_csv import ParseCSVAction
+
+__all__ = [
+    'ParseCSVActionTestCase'
+]
+
+MOCK_DATA_1 = """
+first,last,year
+name1,surename1,1990
+""".strip()
+
+MOCK_DATA_2 = """
+first|last|year
+name1|surename1|1990
+""".strip()
+
+
+class ParseCSVActionTestCase(unittest2.TestCase):
```

Comment:
A `BaseActionTestCase` would be nice :)

Not sure if we have much value to add right now but I suspect we will in the future.